### PR TITLE
AP-8971 - fix various issues related to missing data and rounding

### PR DESCRIPTION
**Pull request by donnen**

### CODE REVIEW

File: classes/apruve_api.php [hunk 1]
```diff
@@ -7,142 +7,158 @@
...
+	 * @param       array $charge_data
+	 *
+	 * @return      array
+	 */
+	public static function create_charge( $charge_data ) {
+		return ApruveWoo_API::post_data( $charge_data );
+	}
+
+	/**
+	 * Post data to Stripe's servers by passing data and an API endpoint
```
Comment (simonbtomlinson) [inline classes/apruve_api.php]: Do we support this at all right now? Stripe stuff seems like it could just be removed.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "require": {
-        "apruve/apruve-php": "~1.1"
+        "apruve/apruve-php": "~1.4",
```
Comment (simonbtomlinson) [inline composer.json]: This'll bump up to 1.5 when we close the other pr, right?

Comment (donnen) [reply composer.json]: I wasn't planning on versioning the package since we are just fixing bugs.

File: classes/apruve_api.php (hunk 1 above)
Comment (donnen) [reply classes/apruve_api.php]: Yeah, I'm not sure what that code is used for.

**Pull request merged**